button-merchant/src/main/java/com/usebutton/merchant/Order.java

```diff
@@ -358,7 +358,96 @@ public Builder setAttributes(Map<String, String> attributes) {
     /**
      * Represents a customer in the order {@link Order#customer}
      */
-    public class Customer {
+    public static class Customer {
 
+        private String id;
+        @Nullable
+        private String email;
+        @Nullable
+        private String emailSha256;
```

Comment:
Why do you have both `email` and `emailSha256`?

Reply:
To allow brands to choose which way they want to give us emails. If we're supplied an email then we will do the hashing but if we're supplied the hash then we just forward that to the api